float32 x_magnetic
float32 y_magnetic
float32 z_magnetic
